picoquic/frames.c

```diff
@@ -1533,21 +1698,37 @@ int picoquic_decode_connection_close_frame(picoquic_cnx_t* cnx, uint8_t* bytes,
 
     if (l1 == 0 || byte_index > bytes_max || bytes_max < 4) {
         ret = picoquic_connection_error(cnx,
-            PICOQUIC_TRANSPORT_FRAME_ERROR(picoquic_frame_type_connection_close));
+            PICOQUIC_TRANSPORT_FRAME_ERROR(ftype));
```

Comment:
Is the error PICOQUIC_TRANSPORT_FRAME_ERROR returns a valid QUIC error per spec?

Reply:
It is defined in the quic spec as a range, with the final 8 bits encoding the frame type. Or at least it was...